firestore/firestore.rules

```diff
@@ -138,5 +138,12 @@
       allow create: if canManageSurvey(getSurvey(surveyId), getEmail()) || isCreator(request.resource, getEmail());
       allow write: if canManageSurvey(getSurvey(surveyId), getEmail()) || isCreator(resource, getEmail());
     }
+
+    // Apply passlist and survey-level ACLs to job documents.
+    match /surveys/{surveyId}/jobs/{jobId} {
+      allow read: if canViewSurvey(getSurvey(surveyId), getEmail());
```

Comment:
Note the args to getSurvey() and other functions have changed and are no longer required.